Formula/faust.rb

```diff
@@ -0,0 +1,49 @@
+class Faust < Formula
+  desc "Functional AUdio STream is language for signal processing and synthesis."
+  homepage "http://faust.grame.fr"
+
+  url "https://github.com/grame-cncm/faust/archive/v2-1-0.tar.gz"
+  sha256 "9bd0b02020a6d9130ac3e2b11ad7cbfc03883769eb87dcb76251c80c40488494"
+  head "https://github.com/grame-cncm/faust.git", :branch => "faust2"
+
+  option "without-httpd", "don't compile httpdlib (requires GNU libmicrohttpd)"
```

Comment:
Too many options, let's provide a full-featured package and include everything. Non-default options are not tested in our CI, so we try to reduce the number of options to the minimum… which is zero in many cases :)